src/libos/src/misc/rlimit.rs

```diff
@@ -21,16 +21,15 @@ impl Default for ResourceLimits {
         // Get memory space limit from Occlum.json
         let cfg_heap_size: u64 = config::LIBOS_CONFIG.process.default_heap_size as u64;
         let cfg_stack_size: u64 = config::LIBOS_CONFIG.process.default_stack_size as u64;
-        let cfg_mmap_size: u64 = config::LIBOS_CONFIG.process.default_mmap_size as u64;
 
         let stack_size = rlimit_t::new(cfg_stack_size);
 
         // Data segment consists of three parts: initialized data, uninitialized data, and heap.
         // Here we just approximatively consider this equal to the size of heap size.
         let data_size = rlimit_t::new(cfg_heap_size);
         // Address space can be approximatively considered equal to the sum of application's
-        // heap, stack and mmap size.
-        let address_space = rlimit_t::new(cfg_heap_size + cfg_stack_size + cfg_mmap_size);
+        // heap, stack size.
+        let address_space = rlimit_t::new(cfg_heap_size + cfg_stack_size);
```

Comment:
RLIMIT_AS indicates the maximum virtual memory size of the process, which may affect calls such as brk and mmap. Is it reasonable to return the sum of the heap and stack sizes?